zanata-common-api/src/main/java/org/zanata/rest/service/FileResource.java

```diff
@@ -45,6 +48,7 @@
     @Deprecated
     public static final String FILE_RESOURCE = SERVICE_PATH;
     public static final String ACCEPTED_TYPES_RESOURCE = "/accepted_types";
+    public static final String ACCEPTED_TYPE_LIST_RESOURCE = "/accepted_type_list";
```

Comment:
`"/accepted_types"` and `"/accepted_type_list"` sound like they would be exactly the same thing. I think we should try to make the names more different.

We should make `ACCEPTED_TYPES_RESOURCE` `@Deprecated` because it is not well named (it gives extensions, not types). I think we should remove it, but not right away because it would break the current clients.

I do not have any very good suggestion for the new endpoint name. Maybe `"accepted_document_types"` for now, and in a future release after `"/accepted_types"` is removed, we could alias that to point to the new endpoint as well.